src/test/java/org/jitsi/jicofo/bridge/BridgeTest.java

```diff
@@ -0,0 +1,82 @@
+package org.jitsi.jicofo.bridge;
+
+import org.jitsi.jicofo.util.*;
+import org.jitsi.xmpp.extensions.colibri.*;
+import org.junit.*;
+import org.jxmpp.jid.*;
+import org.jxmpp.jid.impl.*;
+import org.jxmpp.stringprep.*;
+
+import static org.jitsi.xmpp.extensions.colibri.ColibriStatsExtension.*;
+
+public class BridgeTest
+{
+    @Test
+    public void getStress()
+        throws
+        XmppStringprepException
+    {
+        // Everything should work regardless of the type of jid.
+        Jid jvb1Jid = JidCreate.from("jvb.example.com");
+        Bridge bridge = new Bridge(null, jvb1Jid, null);
+
+        // In the most balanced scenario, in a 100 people call over 4 bridges,
+        // each bridge handles 25 participants. Note that we can't know in
+        // advance whether a participant will be sendrecv or a recvonly.
+
+        /*
+         * Where do the expected values come from? Are they hard-coded to match
+         * the implementation (as it was in a previous commit)?
```

Comment:
The values come from the implementation. It gives you an idea of how the system will behave in practice and you can adjust the code accordingly.